pkg/storaged/stratis-details.jsx

```diff
@@ -55,6 +56,17 @@ import { std_reply, with_keydesc, with_stored_passphrase, confirm_tang_trust, ge
 
 const _ = cockpit.gettext;
 
+export function check_stratis_warnings(client, enter_warning) {
+    if (!client.features.stratis_grow_blockdevs)
+        return;
+
+    for (const p in client.stratis_pools) {
+        const blockdevs = client.stratis_pool_blockdevs[p] || [];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230823-090550-a4293775-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)